core/env.js

```diff
@@ -85,14 +85,21 @@
   
   /** Get a list of boards that we know about */
   function getBoardList(callback) {
-    Espruino.Core.Utils.getJSONURL(JSON_DIR + "boards.json", function(boards){
+    var jsonDir = Espruino.Config.BOARD_JSON_URL || DEFAULT_JSON_DIR;
```

Comment:
I'm not sure if this is needed? If `Espruino.Core.Config.add("BOARD_JSON_URL", ` is set in boardJSON then you're guaranteed that `Espruino.Config.BOARD_JSON_URL` will contain something useful?